hugo/layouts/partials/oct-footer.html

```diff
@@ -1,11 +1,14 @@
-<footer class="barlow-light bg-violet">
-    <div class="footer-content">
-        <a href="/privacy" class="fg-white">Privacy Policy</a>
+<footer class="w-full mt-[75px] flex justify-center items-center text-white bg-eminence font-barlow font-light">
+    <div class="flex justify-between items-center gap-4 py-4 w-4/5 max-w-[640px]">
+        <a href="http://www.eclipse.org/legal/privacy.php" class="hover:underline">Privacy Policy</a>
+        <p class="my-[1em]"> • </p>
+        <a href="http://www.eclipse.org/legal/termsofuse.php" class="hover:underline">Terms of Use</a>
         <p> • </p>
-        <a href="/tos" class="fg-white">Terms of Use</a>
+        <a href="http://www.eclipse.org/legal/copyright.php" class="hover:underline">Copyright Agent</a>
```

Comment:
According to https://www.eclipse.org/projects/handbook/#trademarks-website-footer there should also be a link to "Legal"